app/controllers/api/accounts_controller.rb

```diff
@@ -6,21 +6,32 @@
 #  https://github.com/puzzle/cryptopus.
 
 class Api::AccountsController < ApiController
+  helper_method :team
 
+  # GET /api/api_users
   def index
     authorize Account
     accounts = current_user.accounts
     accounts = find_accounts(accounts)
     render_json accounts
   end
 
+  # GET /api/api_users/:id
   def show
     account = Account.find(params[:id])
     authorize account
     account.decrypt(decrypted_team_password(account.group.team))
     render_json account
   end
 
+  # PATCH /api/api_users/:id?Query
+  def update
+    account = Account.find(params[:id])
```

Comment:
maybe add a method for getting the account by id ? used in show and update method

```
def account
  Account.find ...
end
```